### PR TITLE
Design teardown features

### PR DESCRIPTION
**Pull request by warblo001**
In this project, we were asked to create a Design Teardown from the webpage smashingmagazine.com using frontend technologies as floats, grids, articles, flexbox, and screen size to 1440px.

The project has the following features:
- Navigation Bar with hovering effects
- Backgrounds features on all the project
- Flexbox, grid, floats, and articles
- Screen size of 1440px

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+              <li class="nav-icons"></li>
+              <li class="nav-icons"></li>
+              <li class="nav-icons"></li>
+              <li class="nav-icons"></li>
+            </ul>
+          </div>
+            <div class="search-bar"></div>
+      </div>
+    </nav>
+
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use **header** tag

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
This is an amazing project. You are done with almost all the requirements. But few changes required to approve this project
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**
- [x] Your license link is broken. Please correct the path. 

- [x] Please use Dry CSS. You have used the same properties again and again like font-size: 40px.

- [x] Please use **header** tag for navbar

- [x] you have properly added content in your [index.html](https://github.com/warblo001/DesignTeardown/blob/Design-Teardown-features/index.html) but your [live link](https://rawcdn.githack.com/warblo001/DesignTeardown/ecc85f3b89fd94a88cd1bd7de6901886246b8ce0/index.html) is not rendering the original content present in index.html. Please remove this bug. 

- [x] Whenever I hover over this section, my cursor disappears. Can you please fix it? 

chrome-capture (47)

- [x] It's great to see that you have understood the requirements of the project but in the navigation bar there are only 6 links, 1 logo, and 1 search bar ut you have too many of them. Please have a look at the original page. 
chrome-capture (48)

- [x] In your clone at footer you have made dots a link but on the original page, these are not links. 
chrome-capture (49)



**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by warblo001** [comment]
Fixed issues in the previous comment, we couldn't find a way to remove the cursor pointer from the dots in the list items, on the navbar we noticed there were 7 items instead of 9, on the Dry CSS we changed the font size into a variable, but we don't know if we should do it for all the repeating things, and how we can spot them. Thanks

**Review by arslanbisharat — APPROVED** [review]
### Congratulations Approved 🎉
---------------------------------------------------------------------------------------------------------------
This project is approved with the latest changes you made in the project. You did a great job by removing links dots and making cursor disabling and all the other changes.  
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**